lib/tasks/asset_manager.rake

```diff
@@ -7,8 +7,8 @@ namespace :asset_manager do
     migrator.perform
   end
 
-  task migrate_attachments: :environment do
-    MigrateAssetsToAssetManager.migrate_attachments
+  task :migrate_attachments, %i(batch_start batch_end) => :environment do |_, args|
+    MigrateAssetsToAssetManager.migrate_attachments(args[:batch_start].to_i, args[:batch_end].to_i)
```

Comment:
Minor: I think this implies the arguments are required. Is that right? If so, it would be nice to add a task description to explain this.

Reply:
Minor: See my previous concern about using `String#to_i`. However, I'm not so bothered about it here.

Reply:
I've added a description and usage guidance if the arguments are not supplied.